club/src/main/java/com/dbs/club/domain/meeting/MeetingService.java

```diff
@@ -45,4 +47,25 @@ public Page<MeetingResponseDto.List> getMeetings(int page) {
 
         return meetings.map(MeetingResponseDto.List::fromEntity);
     }
+
+    @Transactional
+    public  void updateMeeting(MeetingRequestDto.Update request, Long meetingId) {
+        Meeting meeting = getMeeting(meetingId);
+
+        if (!canUpdateMeetingDate(meeting.getDate())) {
```

Comment:
if(meeting.canNotUpdateMeetingDate(LocalDate.now())) {
            throw new MeetingException(ErrorCode.MEETING_CAN_NOT_UPDATE);
        }
        
        
public boolean canNotUpdateMeetingDate(LocalDate now) {
        return this.date.isBefore(now);
    }